static/js/components/JobDetails.jsx

```diff
@@ -190,8 +227,9 @@ var JobDetails = React.createClass({
   },
 
   cronUsage: function() {
+    var msg = "Note: To reduce errors, direct modifications aren't currently supported. Please instead deactivate the old job and add a new.";
```

Comment:
I'm not sure what the ideal solution is here.
IMO: Bottom line is that either the delay for jobs being synced to the cron system should be clear to the user (can we query for that interval?) or the sync interval should be <= 1 minute where it is essentially irrelevant